mdsuite/calculators/calculator.py

```diff
@@ -35,6 +37,10 @@
 from tqdm import tqdm
 from typing import Union, List, Any
 
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
```

Comment:
This always causes me issues. I am not sure we should do it.

Reply:
I understand what you mean, but it helps the type checking and with `from __future__ import annotations` this should be perfectly fine. The downside of having this import looking suspiciouse I think is less important than having the benefti of correct type checking for circular imports.